server/src/main/java/org/elasticsearch/action/support/ActiveShardCount.java

```diff
@@ -164,19 +165,23 @@ public boolean enoughShardsActive(final ClusterState clusterState, final String.
     }
 
     /**
-     * Returns true iff the active shard count in the shard routing table is enough
-     * to meet the required shard count represented by this instance.
+     * Returns a tuple where the first value is true iff the active shard count in the shard routing table is enough
+     * to meet the required shard count represented by this instance, and the second value is the active shard count.
      */
-    public boolean enoughShardsActive(final IndexShardRoutingTable shardRoutingTable) {
+    public Tuple<Boolean, Integer> enoughShardsActive(final IndexShardRoutingTable shardRoutingTable) {
```

Comment:
Since this is a public method, should it return a record? Not sure about naming though. `EnoughShards`?

Reply:
Thanks @idegtiarenko ! Sure, made a record.